### DISCUSSION TITLE
Last Attempted Proof stop to update, fake farming

### DISCUSSION THREAD

**Discussion by comquer77** [Chia-Network/chia-blockchain#6612] [Support]
Chia version 1.17 and 1.16
Windows 10 pro, OS antivirus and Firewall disabled.
100Mb home broadband, with PPPoE Internet connection. 
I have totally 20 desktop PCs, this issue happened randomly among these computers. All these computers running full chia node, plotting and farming in the same time. 
Problem:  GUI shows "Synced and Farming" but after a certain time, normally couple days, the last attempted proof stopped to update. Had no choice but stop plotting, quit and restart the GUI to fix the issue. All the unfinished plot lost.  Checked the Internet connection, okay to browse Internet. Checked the TCP/IP connection on the router， found the TCP/IP connection only 1 or 2 for this host. But for the other host, usually have around 10 TCP/IP connection.  Manually add Full Node, but GUI is freezing to response. Force reload the Chia GUI,  shows "connecting to wallet" forever.  

Help needed!

**Comment by loppefaaret** [comment]
i think you might be overloading your consumer router / and or your modem - generally some of these tend to start buckling already at 250-300 conenctions - and seeing you got 20 computers all running the Chia app, acting as nodes, this is easily reached - perhaps you want to start looking into [Farming on many machines](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines)

**Comment by BrandtH22** [comment]
@comquer77 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
